math/exp.mbt

```diff
@@ -12,6 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub fn exp(input : Double) -> Double {
+trait Exp {
+  exp(Self) -> Double
+}
```

Comment:
what's the expected return type of exp function?
should it be `Self`, `Double` or `Float`?

Reply:
I think `Self` is a proper choose.

If `Int.exp(x)` need a `Double` result, users always can turn it to `Double` first.